app/services/katello/ui_notifications/pulp/proxy_disk_space.rb

```diff
@@ -0,0 +1,49 @@
+module Katello
+  module UINotifications
+    module Pulp
+      class ProxyDiskSpace
+        class << self
+          def deliver!
+            SmartProxy.unscoped.with_content.each do |proxy|
+              percentage = proxy.statuses[:pulp].storage['pulp_dir']['percent']
+              if percentage[0..2].to_i < 90 && notification_already_exists?(proxy)
+                blueprint.notifications.where(subject: proxy).destroy_all
```

Comment:
This will clear all notifications for a proxy I think? Really it should only clear the 'pulp_low_disk_space' notification as there could be other types.

Reply:
@sean797 does the reply from Ohad resolved your question? Is there something required on the contributor side?

Reply:
Yes, I don't think so, thanks!